session.go

```diff
@@ -2659,6 +2675,13 @@ func (p *Pipe) Batch(n int) *Pipe {
 	return p
 }
 
+// MaxTimeMS sets the maximum amount of time to allow the query to run.
+//
+func (p *Pipe) MaxTimeMS(n int64) *Pipe {
```

Comment:
It's probably nicer to expose this as `MaxTime(d time.Duration)` and convert to milliseconds internally.

Reply:
Or `QueryTimeout()`? `SetQueryTimeout()`?

Reply:
Absolutely right, i'm changing the parameter to a `time.Duration`. Regarding the naming, i chose the same name as the parameter just to be no brainer and map it 1 to 1 but it is definetly not a problem, you decide! Tell me what you think it's the best to be consistent with the rest of the codebase.

Reply:
We already have `cluster.SetSyncTimeout()` so maybe `SetQueryTimeout()` - what do you think?

Reply:
Sounds good to me!

Reply:
@domodwyer should be ok now